### DISCUSSION TITLE
[Bug]: Papermill Provider installed via pip seems empty

### DISCUSSION THREAD

**Discussion by deramos** [apache/airflow#39292] [General]
### Apache Airflow Provider(s)

papermill

### Versions of Apache Airflow Providers

``` pip install apache-airflow-providers-papermill ``` installs successfully, but the installed package seems empty 

<img width="384" alt="image" src="https://github.com/apache/airflow/assets/3740968/04b81284-73d6-4cd2-be38-8384361fa92d">

Also, library import on pycharm doesn't have papermill under providers

<img width="738" alt="image" src="https://github.com/apache/airflow/assets/3740968/eab49e40-b043-4c93-a0d7-3fef21388ed3">
 

### Apache Airflow version

2.8.4

### Operating System

macOS Sonoma 14.0 (23A344)

### Deployment

Virtualenv installation

### Deployment details

Python 3.9 virtualenv
Pip 24.0

### How to reproduce

Run 
```pip install apache-airflow ``` on macOS 14.0 (23A344)

Then run 
``` pip install  apache-airflow-providers-papermill```

You'd probably get an empty package

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by Taragolis** [comment]
Airflow Community Providers are [namespace packages](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/) and installed under the `airflow/providers` namespace

**Comment by deramos** [comment]
@Taragolis the message I posted earlier shows that `papermill` doesn't exist in the airflow.providers namespace even after pip install. I think you may have missed it.

**Reply by Taragolis** [reply]
This screenshots doesn't shows that. This shows only meta information about package (folder suffixed with `.dist-info`) this folder never contain package files itself, only meta information you could check where it installed into the _RECORD_ file.

Reproducible steps which shows that everything alright

```console
❯ cd /tmp

❯ python -m venv temp-venv

❯ source temp-venv/bin/activate

❯ pip install "apache-airflow==2.9.0" "apache-airflow-providers-papermill==3.6.2" -q                                                      100%
WARNING: You are using pip version 21.2.4; however, version 24.0 is available.
You should consider upgrading via the '/private/tmp/temp-venv/bin/python -m pip install --upgrade pip' command.

❯ python
Python 3.9.9 (main, Nov  1 2022, 18:58:33) 
[Clang 14.0.0 (clang-1400.0.29.102)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from airflow.providers import papermill
>>> help(papermill)

DESCRIPTION
    # Licensed to the Apache Software Foundation (ASF) under one
    # or more contributor license agreements.  See the NOTICE file
    # distributed with this work for additional information
    # regarding copyright ownership.  The ASF licenses this file
    # to you under the Apache License, Version 2.0 (the
    # "License"); you may not use this file except in compliance
    # with the License.  You may obtain a copy of the License at
    #
    #   http://www.apache.org/licenses/LICENSE-2.0
    #
    # Unless required by applicable law or agreed to in writing,
    # software distributed under the License is distributed on an
    # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    # KIND, either express or implied.  See the License for the
    # specific language governing permissions and limitations
    # under the License.
    #
    # NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE
    # OVERWRITTEN WHEN PREPARING DOCUMENTATION FOR THE PACKAGES.
    #
    # IF YOU WANT TO MODIFY THIS FILE, YOU SHOULD MODIFY THE TEMPLATE
    # `PROVIDER__INIT__PY_TEMPLATE.py.jinja2` IN the `dev/breeze/src/airflow_breeze/templates` DIRECTORY
    #

PACKAGE CONTENTS
    get_provider_info
    hooks (package)
    operators (package)

DATA
    __all__ = ['__version__']

VERSION
    3.6.2

FILE
    /private/tmp/temp-venv/lib/python3.9/site-packages/airflow/providers/papermill/__init__.py
```

About your IDE, that might be miss configured to another virtual environment or just not reindexed, or something else
I do not see any problem if I install it into the same virtualenv which use by PyCharm

![image](https://github.com/apache/airflow/assets/3998685/28db7ac7-ec8e-464e-ab0f-6e7dbb177477)
